root/components/EventsList.js

```diff
@@ -46,12 +46,12 @@ const EventsList = ({
   artistRoles,
   checkboxes,
   events,
-  noArtists,
-  noLocation,
-  noRatings,
-  noType,
   order,
   seriesItemNumbers,
+  showArtists,
+  showLocation,
+  showRatings,
+  showType,
```

Comment:
It probably would've made sense if you kept the previous defaults though (i.e. with `showArtists = true,` etc. here), so that you don't have to specify these everywhere unless they're false. (Not sure if that's true for all of them, but in general it seems better for the default to reflect what's most common.)

Reply:
IMO that's kinda confusing. If we're going to make it default to true and have showArtists={false} or whatever, then we're back to specifically negating parts rather than requesting them, and then having it as noArtists seems to make more sense.

Reply:
Is it? The problem I see with `noArtists` is that double-negatives like `noArtists={false}` are confusing, but I don't think negating a positive statement like `showArtists={false}` is confusing.

If you think it makes sense for all the positive props to default to false then let's do that, though I wonder if that doesn't make the component more annoying to use when you need `showArtists={true}` in 4 out of 5 places.

Reply:
Well, it did make it a bit annoying to convert. But I feel in the future, when using it, it'll be much easier to remember "if I want it to show artists, I should add showArtists here" :)